.gitignore

```diff
@@ -2,3 +2,4 @@ build
 composer.lock
 vendor
 .phpunit.result.cache
+/.idea
```

Comment:
Local setup and not project specific - please use your local global gitignore file.
https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
Yep, I agree!